test/unit/component.spec.js

```diff
@@ -258,18 +258,30 @@ describe("trigger method", function() {
 		//Then
 		expect(eventType).equal("eventType");
 		expect(stopType).equal("function");
-		});
-	
-		it("shouldn't call extended method when the Array is extends", () => {
+	});
+
+	it("shouldn't call extended method when the Array is extends", () => {
 		//Given
 		Array.prototype.ExtendSomthing = sinon.spy();
 		oClass.on("eventType", function() {});
 		//When
 		oClass.trigger("eventType");
 		//Then
 		expect(Array.prototype.ExtendSomthing.called).to.be.false;
+	});
+	delete Array.prototype.ExtendSomthing;
```

Comment:
Why did not you put the delete statement into it scope?
```js
it("shouldn't ...", () => {
   // ...
   expect(Array.prototype.ExtendSomthing.called).to.be.false;
   delete Array.prototype.ExtendSomthing;
});
```

Reply:
👍  Thanks. I going to update it.